packages/airnode-protocol/package.json

```diff
@@ -29,7 +29,7 @@
     "write-example-env-file": "hardhat run scripts/write-example-env-file.ts"
   },
   "devDependencies": {
-    "@api3/chains": "^4.1.1",
+    "@api3/chains": "^4.3.0",
```

Comment:
Last thing (I'm pretty sure 😅) - can you make the same change to the `package.json` in `airnode-node` then rerun the yarn commands so there is a single version of `@api3/chains` in the monorepo?

Reply:
Ah, probably missed that. Just updated it.